src/main.cpp

```diff
@@ -490,6 +492,12 @@ int init_options(int argc, char**argv, struct options& options)
     }
   }
 
+  if (!diameter_timeout_set)
```

Comment:
This makes more sense with the other config validation steps at https://github.com/Metaswitch/homestead/blob/dev/src/main.cpp#L632 surely?